<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Pavel Ivanov</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>379</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>614</width>
    <height>379</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>614</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CME Parser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>543</width>
     <height>164</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="terminalSetup">
      <item>
       <layout class="QHBoxLayout" name="timeZoneSetup"/>
      </item>
      <item>
       <widget class="QGroupBox" name="symbolTableGroup">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>200</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>333</width>
          <height>266</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>333</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Символы</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="addSymbolItemBtn">
               <property name="minimumSize">
                <size>
                 <width>86</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Добавить</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/add.png</normaloff>icons/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeSymbolItemBtn">
               <property name="minimumSize">
                <size>
                 <width>86</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Удалить</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/delete.png</normaloff>icons/delete.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dbApplyBtn">
               <property name="minimumSize">
                <size>
                 <width>86</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>86</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>Применить</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>icons/accept_button.png</normaloff>icons/accept_button.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="symbolTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ContiguousSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <row>
              <property name="text">
               <string>1</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>2</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>Имя в CME</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Имя в MT4</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>EUU</string>
              </property>
             </item>
             <item row="0" column="1">
              <property name="text">
               <string>EURUSD</string>
              </property>
             </item>
             <item row="1" column="0">
              <property name="text">
               <string>ADU</string>
              </property>
             </item>
             <item row="1" column="1">
              <property name="text">
               <string>EURAUD</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="parserControl">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>257</width>
        <height>136</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>136</height>
       </size>
      </property>
      <property name="title">
       <string>Парсер</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>26</y>
         <width>233</width>
         <height>84</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="refreshLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Время обновления страницы, сек.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="refreshTimeSpin">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>3600</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="enterCmeBtn">
            <property name="text">
             <string>Войти в CME</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getInfoBtn">
            <property name="toolTip">
             <string>Нажать после входа</string>
            </property>
            <property name="text">
             <string>Нажать после входа</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="startParserBtn">
          <property name="text">
           <string>Начать</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="logText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>addSymbolItemBtn</tabstop>
  <tabstop>removeSymbolItemBtn</tabstop>
  <tabstop>symbolTable</tabstop>
  <tabstop>refreshTimeSpin</tabstop>
  <tabstop>enterCmeBtn</tabstop>
  <tabstop>startParserBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
